pkg/releases/releases.go

```diff
@@ -54,6 +54,9 @@ func (r ReleasesInfo) String() string {
 		case "php":
 			pkgID = "Composer"
 			pkgURL = fmt.Sprintf("https://packagist.org/packages/%s#v%s", info.PackageName, info.Version)
+		case "terraform":
```

Comment:
there should be a test file here that can be updated to make sure this works as expected

Reply:
done